packages/datastore/datastore/src/react/hooks/query.ts

```diff
@@ -19,10 +19,10 @@ const createSubscribeToMore = (model: Model, dispatch: Dispatch<Action>) => {
     };
 };
 
-const forceDelta = async (model: Model, state: ResultState, dispatch: Dispatch<Action>,  options?: IQueryOptions) => {
+const forceDelta = async (model: Model, state: ResultState, dispatch: Dispatch<Action>, options?: IQueryOptions) => {
     if (options?.forceDelta && model.replication && !state.isDeltaForced) {
-      // TODO
-      // await model.replicator.forceDeltaQuery();
+        // TODO
+        // await model.replicator.forceDeltaQuery();
```

Comment:
Ah.. I have changed that. Going to be ugly conflicts here ;D

Reply:
I didn't change anything here, I formatted the file so the spacing changed. When you merge your PR, I'll rebase this branch

Reply:
Ah.. no worries. I will resolve my conflicts.